dockerfile-image-update/src/main/java/com/salesforce/dockerfileimageupdate/CommandLine.java

```diff
@@ -83,7 +83,7 @@ static ArgumentParser getArgumentParser() {
                 .setDefault(false)
                 .help("Only update image tag store. Skip creating PRs");
         parser.addArgument("-x")
-                .help("comment snippet after FROM instruction for ignoring a child image. " +
+                .help("comment snippet mentioned in next line after FROM instruction for ignoring a child image. " +
```

Comment:
I'm begrudgingly ok with this. It seem more intuitive for users to have:
```
# no-dfiu
FROM xyz:123
```

vs. 
```
FROM xyz:123
# no-dfiu
```

Reply:
@afalko : Changed the handling as suggested.

> # no-dfiu
>FROM xyz:123

Reply:
Thanks, this is great